src/patternfly/components/OverflowMenu/overflow-menu.scss

```diff
@@ -0,0 +1,141 @@
+$pf-c-overflow-menu--breakpoint-map: build-breakpoint-map("base", "md", "lg", "xl");
+
+.pf-c-overflow-menu {
+  // Base spacer - shared value
+  --pf-c-overflow-menu--spacer--base: var(--pf-global--spacer--md);
```

Comment:
Do you need to include base here? What does base represent exactly?

Reply:
It's a shared, base spacer value the is used by `__group` and `__item`. It's basically a hook for theming.